docs/source/getting-started.rst

```diff
@@ -207,8 +207,8 @@ a Slack channel using a bot when a monitor fails.
             SendSlackMessageSpiderFinished,
         ]
 
-After enabling the action, you need to provide the `Slack credentials`_ in your `settings.py`
-file as follows:
+After enabling the action, you need to provide the `Slack
+credentials`_. You can access the required credentials by following these :ref:`steps`. Later, fill the same in your `settings.py` as follows:
```

Comment:
There’s still a ‘steps’ reference here.

Reply:
Corrected.

Reply:
Do check, out the sentence after render. Can't do anything about the capital though

![Screenshot_2019-08-29_20-44-34](https://user-images.githubusercontent.com/22801822/63952872-ec9e8a00-ca9d-11e9-8099-0a388f117c6e.png)